presqt/api_v1/helpers/function_router.py

```diff
@@ -14,6 +15,10 @@ class FunctionRouter(object):
     Target Resource Detail:
         {target_name}_detail
 
+    Target Resource Download:
+        {target_name}_download
+
     """
     osf_resource_collection = osf_fetch_resources
-    osf_resource_detail = osf_fetch_resource
\ No newline at end of file
+    osf_resource_detail = osf_fetch_resource
+    osf_resource_download = osf_download_resource
```

Comment:
Hey - I can show you a trick on how you can implement getattr() in this class so that you don't have to call it manually elsewhere. It's pretty neat... ;)

Reply:
Think about making a classmethod instead of using getattr